### PR TITLE
rbd: log LastUpdate in UTC format

### PR DESCRIPTION
**Pull request by Madhu-1**
This PR converts the LastUpdate from int to the UTC format and logs it for better debugging.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@raghavendra-talur JFYI we can return the LastUpdate UTC time in `ResyncVolumeResponse` in the next version.

File: internal/rbd/replicationcontrollerserver.go [hunk 1]
```diff
@@ -614,13 +615,14 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 			return nil, status.Error(codes.Internal, err.Error())
 		}
 	}
-
+	// convert the last update time to UTC
```
Comment (nixpanic) [inline internal/rbd/replicationcontrollerserver.go:619]: nit: empty lines between sections of a function make it more readable, it was not needed to remove it.

**Review by nixpanic — DISMISSED** [review]

File: internal/rbd/replicationcontrollerserver.go (hunk 1 above)
Comment (Madhu-1) [reply internal/rbd/replicationcontrollerserver.go:619]: Brought it back.

**Review by nixpanic — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Comment by Rakshith-R** [comment]
> /retest ci/centos/upgrade-tests-cephfs

failed due to https://github.com/ceph/ceph-csi/issues/2264
[logs](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/upgrade-tests-cephfs/detail/upgrade-tests-cephfs/2602/pipeline)

**Comment by nixpanic** [comment]
> /retest ci/centos/upgrade-tests-cephfs

```
Aug  6 06:52:15.703: INFO: Running '/usr/bin/kubectl --server=https://192.168.39.32:8443 --kubeconfig=/root/.kube/config --namespace=cephcsi-e2e-a4872b0f create -f -'
Aug  6 06:52:24.884: INFO: rc: 1
Aug  6 06:52:24.884: INFO: will run kubectl (create) again (20 seconds elapsed)
Aug  6 06:52:24.884: FAIL: failed to create CSIDriver object with error failed to run kubectl: error running /usr/bin/kubectl --server=https://192.168.39.32:8443 --kubeconfig=/root/.kube/config --namespace=cephcsi-e2e-a4872b0f create -f -:
Command stdout:

stderr:
Error from server (AlreadyExists): error when creating "STDIN": csidrivers.storage.k8s.io "cephfs.csi.ceph.com" already exists

error:
exit status 1
```
should be fixed with #2329

**Review by humblec — APPROVED** [review]

**Pull request merged**